packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorAccountTabs.jsx

```diff
@@ -144,4 +216,4 @@ KonnectorAccountTabs.propTypes = {
   innerAccountModalOverrides: innerAccountModalOverridesProptype
 }
 
-export default KonnectorAccountTabs
+export default withClient(KonnectorAccountTabs)
```

Comment:
Why are we adding an HOC here? Can't we use useClient() inside the function?

Reply:
Now I use useClient